force-app/main/default/lwc/geBatchWizard/geBatchWizard.js

```diff
@@ -105,6 +109,36 @@ export default class geBatchWizard extends NavigationMixin(LightningElement) {
         return Settings.canMakeGiftsRecurring();
     }
 
+    handleAllowRecurringDonationsOnChange(event) {
+        const isChecked = event.detail.value
+        this._allowRecurringDonations = isChecked;
+        if (!isChecked) {
+            this._allowFirstInstallment = false;
+            this._allowFirstInstallmentDisabled = true;
+        } else {
+            this._allowFirstInstallmentDisabled = false;
+        }
+
+    }
+
+    get allowFirstInstallment () {
```

Comment:
This looks intentional, but when I "Edit Batch Info" and deselect "Allow Recurring Donations", "First Installment Paid" remains checked. I see the extra logic here, but why is this different behavior from the Batch creation wizard?

Reply:
Going to look into this, the idea here is, the `Allow Recurring Donations` checkbox controls the `First Installment Paid` checkbox, if it's unchecked, the other should be unchecked and disabled.